envbuilder.go

```diff
@@ -112,10 +112,16 @@ type Options struct {
 	// It will override CacheRepo if both are specified.
 	LayerCacheDir string `env:"LAYER_CACHE_DIR"`
 
-	// DevcontainerJSONPath is a relative or absolute path to a
-	// devcontainer.json file. This can be used in cases where
-	// one wants to substitute an edited devcontainer.json file
-	// for the one that exists in the repo.
+	// DevcontainerDir is a relative path to the folder containing
```

Comment:
Any reason this can't accept an absolute path as well? Supporting both would make it consistent with `DEVCONTAINER_JSON_PATH`.